app/helpers/services_helper.rb

```diff
@@ -27,7 +27,7 @@ def show_mappings?
   end
 
   def path_to_service(service)
```

Comment:
plz remove this method and use directly `admin_service_integration_path(service)` instead

Reply:
Not necessarily in this same PR 👍🏼